File: selectionsort.py
def selection_sort(list):
    for i in range(len(list) - 1):
        lowest_number_index = i

        for j in range(lowest_number_index + 1, len(list)):
            if list[j] < list[lowest_number_index]:
                lowest_number_index = j

        if (lowest_number_index != i):
            temp = list[i]
            list[i] = list[lowest_number_index]
            list[lowest_number_index] = temp

    return list


print(selection_sort([333, 17, 753, 80, 202]))

File: selectionsort.js
function selectionSort(list) {
  for (let i = 0; i < list.length -1; i++) {
    let lowestNumberIndex = i;
    for (let j = i + 1; j < list.length; j++) {
      if (list[j] < list[lowestNumberIndex]) {
        lowestNumberIndex = j;
      }
    }
    if (lowestNumberIndex != i) {
      let temp;
      temp = list[i];
      list[i] = list[lowestNumberIndex];
      list[lowestNumberIndex] = temp;
    }
  }
  return list
}

console.log(selectionSort([333, 17, 753, 80, 202]))